operator/version/version.go

```diff
@@ -22,6 +22,8 @@ import (
 const (
 	// OperatorCodeBaseVersion is the version string from the code base.
 	OperatorCodeBaseVersion = "1.17.0"
+	// OperatorKubernetesMinSupportedVersion is the minimum supported Kubernetes version
+	OperatorKubernetesMinSupportedVersion = "1.22.0"
```

Comment:
we have `k8sversion.MinK8SVersion` already, can we make sure we just have 1 so we don't have to update multiple places

Reply:
fixed

Reply:
please check again @howardjohn